docs/display/difference.md

```diff
@@ -21,22 +21,33 @@ date1.diff('2018-06-05', 'month') // 7
 
 Units are case insensitive, and support plural and short forms.
 
+```js
+const date1 = dayjs('2020-12-25')
+date1.diff('2020-12-24', 'days') // 1
+```
+
+```js
+const date1 = dayjs('2020-12-25T23:56:25.351Z')
+date1.diff('2020-12-25T20:56:25.351Z', 'h') // 3
+```
+
+
```

Comment:
Thanks, one question here, seems we don't need these code?

Reply:
I actually missed that units are case insensitive, and support plural and short forms the first time I read the documentation; I thought it would be nice to have examples so it's easier to scan the documentation. I can remove this example if you think it's extraneous.

Reply:
I do think this is not a recommended practice to pass in a unit that is plural or short forms. The reason our lib supports it is just for maximum compatibility.